fenix/app/src/main/java/org/mozilla/fenix/debugsettings/tabs/TabTools.kt

```diff
@@ -320,46 +318,14 @@ private class TabToolsPreviewParameterProvider : PreviewParameterProvider<TabToo
 private fun TabToolsPreview(
     @PreviewParameter(TabToolsPreviewParameterProvider::class) model: TabToolsPreviewModel,
 ) {
-    val snackbarState = remember { SnackbarHostState() }
-    val scope = rememberCoroutineScope()
-
-    with(model) {
-        FirefoxTheme {
-            Box(
-                modifier = Modifier.background(color = FirefoxTheme.colors.layer1),
-            ) {
-                TabToolsContent(
-                    activeTabCount = activeTabCount,
-                    inactiveTabCount = inactiveTabCount,
-                    privateTabCount = privateTabCount,
-                    totalTabCount = totalTabCount,
-                    inactiveTabsEnabled = inactiveTabsEnabled,
-                    onAddTabsToActiveClick = {
-                        scope.launch {
-                            snackbarState.showSnackbar("Add $it tabs to active")
-                        }
-                    },
-                    onAddTabsToInactiveClick = {
-                        scope.launch {
-                            snackbarState.showSnackbar("Add $it tabs to inactive")
-                        }
-                    },
-                    onAddTabsToPrivateClick = {
-                        scope.launch {
-                            snackbarState.showSnackbar("Add $it tabs to private")
-                        }
-                    },
-                )
-
-                SnackbarHost(
-                    hostState = snackbarState,
-                    modifier = Modifier.align(Alignment.BottomCenter),
-                ) { snackbarData ->
-                    Snackbar(
-                        snackbarData = snackbarData,
-                    )
-                }
-            }
+    FirefoxTheme {
+        Box(
+            modifier = Modifier.background(color = FirefoxTheme.colors.layer1),
+        ) {
+            TabTools(
+                store = BrowserStore(),
+                inactiveTabsEnabled = model.inactiveTabsEnabled,
+            )
```

Comment:
now that the store action for the tab creation has been plumbed in, the preview logic was able to be considerably condensed.